client/src/assets/scss/_footer.scss

```diff
@@ -0,0 +1,15 @@
+footer {
+  position: fixed;
+  bottom: 0;
+  background-color: rgb(90, 86, 86);
```

Comment:
Color is nice, but we select something more bright

Reply:
agree, let's do yellow